test/v3/connectors/SubmitFinalDeclarationConnectorSpec.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v3.connectors
+
+import mocks.{MockAppConfig, MockHttpClient}
+import uk.gov.hmrc.http.HeaderCarrier
+import v3.models.domain.{EmptyJsonBody, Nino, TaxYear}
+import v3.models.outcomes.ResponseWrapper
+import v3.models.request.SubmitFinalDeclarationRequest
+
+import scala.concurrent.Future
+
+
+class SubmitFinalDeclarationConnectorSpec extends ConnectorSpec {
+
+  val nino: String           = "AA111111A"
+  val taxYear: TaxYear        =  TaxYear("2021-22")
```

Comment:
```suggestion
  val taxYear: TaxYear        =  TaxYear.fromMtd("2021-22")
```
The test only works as is because it replicates what the connector code does and calls `taxYear.toDownstream` - both will evaluate to "2021-22" which is the MTD format.